src/cryptoadvance/specter/hwi_rpc.py

```diff
@@ -0,0 +1,258 @@
+from flask import Flask, url_for, render_template, jsonify, request, make_response, redirect, Response, current_app
+import json, subprocess, time, traceback
+from hwilib.serializations import PSBT
+import hwilib.commands as hwi_commands
+from hwilib import bech32
+from .helpers import normalize_xpubs, convert_xpub_prefix
+
+from .specter_hwi import SpecterClient, enumerate as specter_enumerate
+
+class JSONRPC:
```

Comment:
let's also move `JSONRPC` class to a separate file, we may reuse it later for non-HWI tasks

Reply:
Makes sense, I separated it.